hpcbench/driver.py

```diff
@@ -320,6 +320,10 @@ def __init__(self, parent, tag):
     @cached_property
     def sbatch_args(self):
         sbatch_options = self.campaign.process.get('sbatch', {})
+        if self.tag in self.campaign.benchmarks:
+            tag_sbatch_opts = self.campaign.benchmarks[
+                self.tag].get('sbatch', dict())
+            sbatch_options.update(tag_sbatch_opts)
```

Comment:
Shouldn't we handle this dictionary like in `SrunExecutionDriver._make_srun_arguments` instead of merging it?

Reply:
hmm.. yes, and actually now i see an elegant way to also handle #160